chainbase/src/main/java/org/tron/core/capsule/TransactionCapsule.java

```diff
@@ -251,7 +230,7 @@ public static long checkWeight(Permission permission, List<ByteString> sigs, byt
     } catch (ContractValidateException | InvalidProtocolBufferException e) {
       logger.debug(e.getMessage(), e);
     }
-    return null;
```

Comment:
the class of TransactionUtil has a similar function. Please modify it and check the function where invoked to determine the return value whether null or new byte[0].